include/swift/SIL/SILConstants.h

```diff
@@ -147,15 +146,15 @@ class SymbolicValue {
     AggregateSymbolicValue *aggregate;
   } value;
 
-  union {
+  struct {
     UnknownReason unknown_reason;
 
-    //unsigned integer_bitwidth;
+    llvm::SmallVector<SourceLoc, 4> callStack;
```

Comment:
SymbolicValue is designed to be a POD type that can be thrown around without running its destructor, and sizeof(SymbolicValue) needs to remain small (lots of them get put in densemaps etc).

Please make the callstack be a new tail allocated node, similar to how the apint represent or the aggregate value representation is stored.  We can then have two "unknown" representations, one that has a stack trace (which is seperately allocated and one with an inline representation).